gnat2goto/driver/driver.adb

```diff
@@ -284,4 +314,27 @@ package body Driver is
       end loop;
    end Translate_Standard_Types;
 
+   procedure Add_CProver_Internal_Symbols is
+      procedure Add_Global_Sym (Name : Symbol_Id; Sym_Type : Irep);
+      procedure Add_Global_Sym (Name : Symbol_Id; Sym_Type : Irep) is
+         Sym : constant Symbol :=
+           (Name => Name,
+            BaseName => Name,
+            PrettyName => Name,
+            SymType => Sym_Type,
+            Module => Intern (""),
+            Mode => Intern ("C"),
```

Comment:
What is the symbol mode, I can't remember? Is it a hack for now to have the mode set as `"C"`, and do we plan to introduce an `"ADA"` mode, or will it be like this forever?

Reply:
TBH I don't know what the mode is even good for, but as it's a cprover internal variable it should behave the same way regardless of the source language I think.

Reply:
There's also modes for C, C++, and Java (`ID_C`, `ID_cpp`, `ID_java`). AFAIK they are used to produce language-specific output when e.g. printing the symbol table or the goto program. I think we can use `"C"` for now until we run into an actual situation where we would need an ADA mode.

Reply:
Very useful, thanks @danpoe